src/url-utils.ts

```diff
@@ -1,10 +1,29 @@
 import { resolve } from 'url';
 
-const getUrl = (base: string, projectName?: string): string => {
+const getUrl = (
+  base: string,
+  projectName?: string,
+  namePrefix?: string,
+  tags?: Array<string>,
+): string => {
+  const url = resolve(base, './client/features');
+  const params = new URLSearchParams();
   if (projectName) {
-    return resolve(base, `./client/features?project=${projectName}`);
+    params.append('project', projectName);
   }
-  return resolve(base, './client/features');
+  if (namePrefix) {
+    params.append('namePrefix', namePrefix);
+  }
+  if (tags) {
+    tags.forEach((tag) => {
+      params.append('tag', tag);
```

Comment:
I think we should do a `encodeURIComponent` like this:

`params.append('tag', encodeURIComponent(tag));`